packages/headless/src/features/folding/folding-slice.ts

```diff
@@ -2,8 +2,11 @@ import {createReducer} from '@reduxjs/toolkit';
 import {Result} from '../../api/search/search/result';
```

Comment:
Before reviewing this file, I recommend looking at [this comment](https://github.com/coveo/ui-kit/pull/824/files#r634713699).